File: test/python/fork.py
import os, time
for i in range(0,2):
	result = os.fork()
	if result == 0:
		print "child process", "i:",i , "---ppid : ", os.getppid(), "pid : ", os.getpid()
		time.sleep(2)
	elif result > 0:
		print "parent process", "i:", i, "---ppid : ", os.getppid(), "pid : ", os.getpid()
		time.sleep(2)

File: test/unix/fork.c
//这个可以和缓冲区一起总结
#include<stdio.h>
#include<unistd.h>
int main(){
	for(int i=0; i<2; i++){
		int result = fork();
		printf("ppid : %d\n", getppid());
		printf("& %d  ", getpid());
		/*if(result == 0){
			printf("child process -----");
		}else if(result > 0){
			printf("parent process: child id = %d---------", result);	
		}
		printf("ppid : %d---pid : %d---i=%d\n", getppid(), getpid(), i);*/
	}
	printf("\n");
	sleep(20);
}
